shot-android/src/main/java/com/karumi/shot/compose/ComposeScreenshot.kt

```diff
@@ -10,9 +11,14 @@ class ComposeScreenshot(
     private val saver: ScreenshotSaver
 ) {
 
+    fun saveScreenshot(bitmap: Bitmap, data: ScreenshotMetadata) {
```

Comment:
Should this class be renamed and moved into a different package? I'm also wondering if this generic functionality for screenshots should be separated from `Compose` specific logic.

Reply:
We can leave it as is for now. I don't want to extract it to another class when we don't know how this is going to evolve. As this is not part of the public API, it is ok if we just leave it here and we change it in any future refactor.